wgpu-core/src/command/bundle.rs

```diff
@@ -675,13 +708,16 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         id_in: Input<G, id::RenderBundleId>,
     ) -> Result<id::RenderBundleId, RenderBundleError> {
         span!(_guard, INFO, "RenderBundleEncoder::finish");
+        let err_ctx = PassErrorScope::Bundle;
```

Comment:
can we mass-rename this to `err_scope` or just `scope`?

Reply:
Ah sure, that would make sense in combination with the other change :)